### ISSUE TITLE
[3-layers] Move service part4

### ISSUE THREAD

**Issue by antonbakalets** [ita-social-projects/GreenCity#1616]
### Description

- Move service interfaces 
`HabitFactTranslationService HabitFactService HabitAssignService GraphService GoalService FileService FavoritePlaceService FactOfTheDayTranslationService FactOfTheDayService EcoNewsService`
to `service-api` module
- Move corresponding implementations classes to service module
- Create required DTOs
- Move corresponding unit tests

Already done:  `GoalService GraphService` 

### Acceptance Criteria

- In pom.xml try to mark module `dao` and module `service` with `<scope>runtime</scope>`
- Make sure classes related to your services are at appropriate levels

**Blocked by**
Depends on #1448 and #1449

**Epic link**
Epic #1446
